scalacheck/src/main/scala/pathy/scalacheck/RandomSeg.scala

```diff
@@ -28,9 +28,10 @@ private[scalacheck] object RandomSeg {
   implicit val randomSegArbitrary: Arbitrary[RandomSeg] =
     Arbitrary {
       Gen.nonEmptyListOf(Gen.frequency(
-        100 -> Arbitrary.arbitrary[Char],
+        100 -> Gen.alphaNumChar,
```

Comment:
This is the reason @wemrysi and I introduced the `PathOf` construct so that you can say `PathOf[AlphaCharacters]`, but I think the default should remain a truly arbitrary path. Also, I defined `AlphaCharacters` in quasar, might make sense to pull it into pathy as it seems like a generally useful thing.